15.0/entrypoint.sh

```diff
@@ -10,22 +10,36 @@ fi
 # and pass them as arguments to the odoo process if not present in the config file
 : ${HOST:=${DB_PORT_5432_TCP_ADDR:='db'}}
 : ${PORT:=${DB_PORT_5432_TCP_PORT:=5432}}
+: ${NAME:=${DB_ENV_POSTGRES_NAME:=${POSTGRES_NAME:='postgres'}}}
 : ${USER:=${DB_ENV_POSTGRES_USER:=${POSTGRES_USER:='odoo'}}}
 : ${PASSWORD:=${DB_ENV_POSTGRES_PASSWORD:=${POSTGRES_PASSWORD:='odoo'}}}
 
+ODOO_ARGS=()
 DB_ARGS=()
+
 function check_config() {
     param="$1"
     value="$2"
+    pg_flag="$3"
+
     if grep -q -E "^\s*\b${param}\b\s*=" "$ODOO_RC" ; then       
         value=$(grep -E "^\s*\b${param}\b\s*=" "$ODOO_RC" |cut -d " " -f3|sed 's/["\n\r]//g')
     fi;
-    DB_ARGS+=("--${param}")
-    DB_ARGS+=("${value}")
+
+    ODOO_ARGS+=("--${param}")
+    ODOO_ARGS+=("${value}")
+
+    # Only add to DB_ARGS if pg_flag is set
+    if [[ -n "$pg_flag" ]]; then
+        DB_ARGS+=("${pg_flag}")
+        DB_ARGS+=("${value}")
+    fi
 }
-check_config "db_host" "$HOST"
-check_config "db_port" "$PORT"
-check_config "db_user" "$USER"
+
+check_config "db_name" "$NAME" "-d"
```

Comment:
In order to support "permissive multi-tenant", --db_name cannot become a requirement for ODOO_ARGS.

Reply:
@amh-mw - which solution is preferable:

1. Put a conditional to check config only if `NAME` (`db_name`) is not `postgres`
2. Revert back to hard-coding `postgres`
3. Another solution

With this added context, I agree with your note in #358 that it is likely ok to hardcode this here - but I want to understand whether there is a benefit (such as logging within Postgres) to checking against the DB name as configured.

If not, and if there is no alternate suggestion, I will revert the `NAME` variable addition here :)

Thank you for the added context.